Chaincase.UI/ViewModels/ConnectNodeViewModel.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using Chaincase.Common;
+using ReactiveUI;
+using WalletWasabi.Blockchain.Analysis.FeesEstimation;
+using Chaincase.Common.Services;
+using System.Net;
+using WalletWasabi.Exceptions;
+using System.Globalization;
+using System.Threading;
+using NBitcoin.Protocol;
+using NBitcoin.Protocol.Behaviors;
+
+namespace Chaincase.UI.ViewModels
+{
+    public class ConnectNodeViewModel : ReactiveObject
+    {
+        private readonly Global _global;
+        private readonly ChaincaseWalletManager _walletManager;
+        private readonly Config _config;
+        private readonly UiConfig _uiConfig;
+        private readonly FeeProviders _feeProviders;
+        private readonly ChaincaseSynchronizer _synchronizer;
+
+        private string _nodeAddress;
+        private bool _hasSetCustomAddress;
+        private bool? _canConnected;
+
+        public ConnectNodeViewModel(Global global, ChaincaseWalletManager walletManager, Config config, ChaincaseSynchronizer synchronizer)
+        {
+            _global = global;
+            _walletManager = walletManager;
+            _config = config;
+            _synchronizer = synchronizer;
+            _hasSetCustomAddress = _config.GetBitcoinP2pEndPoint().ToString() != "127.0.0.1:8333";
```

Comment:
do not hardcode the value -- it already causes a mismatch based on being on testnet by default in this PR